foundry/foundry.py

```diff
@@ -761,21 +722,21 @@ def _load_data(self, file=None, source_id=None, as_hdf5=False):
                     path_to_file
                 )
             except Exception as e:
-                print(f"Reading {file} as JSON failed: {e} \n", "Now attempting to read as JSONL")
+                logging.log(logging.WARNING, f"Reading {file} as JSON failed: {e} \n", "Now attempting to read as JSONL")
```

Comment:
small note -- I think it would be better to say `"Cannot read {file} as JSON: {e} \n", "Now attempting to read as JSONL"` since seeing the word "failed" is a little misleading (and concerning!)

Reply:
ditto the following, similar except blocks

Reply:
I think WARNING is overblown. Warnings imply someone should do something about how they are using the code, and them providing JSON is no worse or better than using CSV. Use INFO or, even, DEBUG instead.

Reply:
FWIW, I think this whole block needs some TLC #224

Reply:
Agreed on both accounts! INFO or DEBUG would be much better (probably INFO in this case)

Reply:
@Aadit-Ambadkar this still needs attention